cmd/ocm/list/version/cmd.go

```diff
@@ -55,6 +56,13 @@ func init() {
 		"stable",
 		"List only versions from the specified channel group",
 	)
+
+	fs.StringVar(
+		&args.gcpMarketplaceEnabled,
+		"gcp-marketplace-enabled",
```

Comment:
this is a long name for the flag. Maybe `gcp-marketplace` or `marketplace-gcp` (to match the subscription type value) would be enough?

Reply:
As per the suggestion (https://redhat-internal.slack.com/archives/C05BJQLPYQ5/p1695828401018519?thread_ts=1695762022.253959&cid=C05BJQLPYQ5), changed the flag to gcp-marketplace